python/lsst/ctrl/execute/libexec/qdelete.py

```diff
@@ -41,3 +42,7 @@
     )
     exitCode = cmd.runCommand(command)
     sys.exit(exitCode)
+
+
+if __name__ == "__main__":
```

Comment:
I see that this code is required to exist because the tests try to run the executable commands. This seems less than ideal to me. Can't we assume that by the time the test code runs we have created the `bin` scripts and put them in $PATH?

Reply:
This isn't really new code, it's just a reorganization to add an explicit function definition (`main()`) so that the definition of the entry-point can be complete and declarative, e.g., `qdelete:main`.

Reply:
I was talking specifically about the `if __name__` code. It doesn't need to be there because no-one should be trying to run the package file directly.  I was assuming it was there because you changed the tests to use `python -m`.

Reply:
The `if __name__ == "__main__"` pattern is best practice for any python module that will be called as a script, whether it's by shebang, `python script.py` or `python -m pkg.mod.script`. It's just that of these, the latter is preferable.

I changed the test to use `python -m` because the script it's testing may now be referenced by its module path because it's been moved into the package.

Reply:
"no-one should be trying to run the package file directly."

I don't know about that. It seems to be the exact point of these modules. I know that people directly run `allocateNodes.py` but I have no idea about the others.